Content.Server/Doors/Systems/DoorSystem.cs

```diff
@@ -298,6 +297,18 @@ public override void StartOpening(EntityUid uid, DoorComponent? door = null, Ent
         if(lastState == DoorState.Emagging && TryComp<AirlockComponent>(door.Owner, out var airlockComponent))
             airlockComponent?.SetBoltsWithAudio(!airlockComponent.IsBolted());
     }
+
+    protected override void CheckDoorBump(DoorComponent component, PhysicsComponent body)
+    {
+        if (component.BumpOpen)
+        {
+            foreach (var other in PhysicsSystem.GetCollidingEntities(body))
+            {
+                if (Tags.HasTag(other.Owner, "DoorBumpOpener") &&
+                    TryOpen(component.Owner, component, other.Owner)) break;
+            }
+        }
+    }
```

Comment:
When I had this in shared client would think it could open the door so you would get the open sound spammed in between.

Reply:
Predicted access when